### ISSUE TITLE
Add "drugs" for recycling containers

### ISSUE THREAD

**Issue by Mannivu** [streetcomplete/StreetComplete#4802]
Currently, the Recycling container quest doesn't provide an answer for containers that recycle expired drugs. I think this should be added.
![screenshot showing the lack of "drugs" answer](https://user-images.githubusercontent.com/42747216/218156956-92065271-8a38-4afd-8204-7f42b7151633.jpg)

**Comment by westnordost** [comment]
Why do you think it should be added?

**Comment by Mannivu** [comment]
Because it doesn't fall in any of the other macro-categories shown at the moment

**Comment by westnordost** [comment]
Take a look at the wiki page for `recycling:*`. A lot of things do not. Also, they are not macro-categories, see the hint text at the top. If something is recycling only newspapers, then selecting "paper" is wrong.

**Comment by matkoniecz** [comment]
1) is anyone even actually recycling drugs rather than safely destroying them?

2) how widely this tagging is used? Is it used on scale comparable to other supported values?

**Comment by michalgwo** [comment]
> 1. is anyone even actually recycling drugs rather than safely destroying them?

I'm not sure, but tag recycling:drugs is more used than [waste=drugs](https://wiki.openstreetmap.org/wiki/Tag%3Awaste%3Ddrugs) (358 vs 52), I guess most people just don't know what they are doing with the disposed drugs, so it can be confusing between these 2 options.

> 2. how widely this tagging is used? Is it used on scale comparable to other supported values?

It's not, but it would be nice to have the possibility to add all of the available values. My proposition is to add another option in "Other Answers..." with a list of all available things to dispose, as it will be a long list, it can be a simple list without pictures, this way it won't be so deterrent and will allow people to be more accurate.

Besides that I would add dialog (just like after clicking on plastic) after clicking on paper with options to choose recycling:paper_packaging, recycling:cardboard, recycling:magazines, recycling:newspapers, and recycling:cartons. Same with Electric appliances, options would be recycling:small_appliances, recycling:small_electrical_appliances (not sure what's the difference between these 2, there is no description on OSM Wiki), recycling:electrical_items

**Comment by Mannivu** [comment]
> 1. is anyone even actually recycling drugs rather than safely destroying them?

In Italy we cannot destroy drugs by ourselves, we must throw them in dedicated bins (like [this one](https://i1.wp.com/www.bastiaoggi.it/wp-content/uploads/2019/09/bidoni-farmaci-scaduti-w.jpg?resize=678%2C381&ssl=1)).

> 2. how widely this tagging is used? Is it used on scale comparable to other supported values?

It's not widely used, but I think that this might be because some users don't know how to tag it. I think that michalgwo proposal is quite good, given that for `recycling:` there are some cases that have a wider use than the ones presented by SC

**Comment by matkoniecz** [comment]
Note that listing every single recyclable category is definitely impossible and at least in some cases you will need to create note.

The question is how popular items should be listed in SC with own icon and translated label.

**Comment by Mannivu** [comment]
> Note that listing every single recyclable category is definitely impossible and at least in some cases you will need to create note.

It's not definitely impossible; other app, such as [EveryDoor](https://user-images.githubusercontent.com/42747216/218412256-b9d073b8-95f8-4ffa-ba36-66087b6aaf62.png), let the user look for a specific value, why SC couldn't?

> The question is how popular items should be listed in SC with own icon and translated label.

If it's based on tag popularity that might have been a weird decision, since tag usage can change. But that's another story.

**Comment by westnordost** [comment]
I am closing this for the following reasons:

1. I find the chance for a bin _made specifically_ to dump drugs to be mapped but only with `amenity=recycling` but **without** `recycling:drugs=yes` to be extremely low. Really, no mapper in their right mind would map such a bin only with `amenity=recycling` because that's the tag used for recycling containers.

2. as mentioned, SC does not support all values, only frequent values and only values that are expected to be found on recycling containers (see above) because every option requires an icon+localized text and more importantly, the user should not be drowned in a lot of fringe options when it is going to be paper, glass etc 99% of the time anyway.

**Comment by Mannivu** [comment]
> 1. I find the chance for a bin made specifically to dump drugs to be mapped and only tagged with amenity=recycling but without recycling:drugs=yes to be extremely low. Really, no mapper in their right mind would add the POI only with amenity=recycling because that's the tag used for recycling containers.

So, if I am driving on a road and I see a recycling bin but can't see what they are for, I shouldn't map it just because I don't know what kind of waste should be recycled there?

> 2. as mentioned, SC does not support all values, only frequent values and only values that are expected to be found on recycling containers (see above) because every option requires an icon+localized text and more importantly, the user should not be drowned in a lot of fringe options when it is going to be paper, glass etc 99% of the time anyway.

This I understand, but some other apps have found a way to let the user map a lot of values and I think SC should to (since is one of the most used app for simple mapping on Android).

**Comment by Mannivu** [comment]
> That's a bit of a made-up example. You do not map while driving. If you are a passenger and you see a bin flying by, you'll certainly not add a POI with amenity=recycling to that now, will you? At most amenity=waste_basket, but really, I find the scenario to add POIs on a passenger seat somewhat unrealistic.

I sometimes do it, and sometimes I take a vocal note to remind me to add a POI I've seen while driving. It's not wrong nor unrealistic.

>SC is the most used app precisely because it is quick and easy to use with no knowledge about the tags being used necessary.

Necessary according to whom? Forcing the user to leave a note instead of providing a simple method to map something doesn't seem like "easy to use with no knowledge". New users might not know what a note is in OSM, but they know what they have in front of their eyes.

**Comment by westnordost** [comment]
Mhm, well, if I found that line of argument convincing, I wouldn't have closed this issue.

**Comment by GioviPen** [comment]
> 1. is anyone even actually recycling drugs rather than safely destroying them?

At least in Italy but probably in the whole Europe is the only way to manage them. I agree that should be added, I’ll suggest the same for “Go Map!!” since every time the only other option is writing the tags manually

**Comment by mnalis** [comment]
**TL;DR:** perhaps https://github.com/openstreetmap/id-tagging-schema/pull/1876 would be good enough solution?

--- 

> At least in Italy but probably in the whole Europe is the only way to manage them. 

Sure, but not really important to OSM tagging

<details>

That is the case with many of `recycling:*` tags which do not actually get recycled (in fact, user on the ground can hardly know what actually happens with things put in different containers). e.g. `recycling:asbestos` is likely not being recycled either, and `recycling:food_waste` is in many cases just being incinerated (as meat and dairy can't be composted), most places accepting household batteries do not really recycle them (although some companies have started to recycle electric car batteries, as it is much more profitable) etc.  

But since collecting containers look and function just like one actually used for recycling, `recycling:` prefix is what OSM crowd ended up using for them.

</details>

> I agree that should be added, I’ll suggest the same for “Go Map!!” since every time the only other option is writing the tags manually

I've opened PR at https://github.com/openstreetmap/id-tagging-schema/pull/1876 which would allow you to add "waste drugs container" (should become available  in StreetComplete's "Things Overlay" if it is accepted in id-tagging-schema), so if you'd like that functionality, feel free to go add thumbs up there.

As for the StreetComplete's Recycling **Quest** itself, see https://github.com/streetcomplete/StreetComplete/issues/4802#issuecomment-1427863642 why it's probably not worth it (or is your experience different, i.e. do you get that quest asked often on drugs-only containers, and have to leave an OSM note? can you link to few of them?)
